File: Python/2159.py
# Problema: 2159 - Número Aproximado de Primos  | Resposta: Accepted
# Linguagem: Python 3.11 [+1s]                  |Tempo: 0.000s

from math import log

n = int(input())
minimo = n / log(n)
maximo = 1.25506*minimo
print(f"{minimo:.1f} {maximo:.1f}")

File: JavaScript/2159.js
// Problema: 2159 - Número Aproximado de Primosa | Resposta: Accepted
// Linguagem: JavaScript 12.18 [+2s]             | Tempo: 0.036s

var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var line = input.trim();

let n = parseInt(line);
let min = n / Math.log(n);
let max = 1.25506*min;
console.log(min.toFixed(1), max.toFixed(1));
